src/lib.rs

```diff
@@ -87,6 +87,15 @@ impl Build {
             _ if target.contains("freebsd") => {
                 config.define("LUA_USE_LINUX", None);
             }
+            _ if target.contains("netbsd") => {
+                config.define("LUA_USE_LINUX", None);
+            }
+            _ if target.contains("openbsd") => {
+                config.define("LUA_USE_LINUX", None);
+            }
+            _ if target.contains("bitrig") => {
+                config.define("LUA_USE_LINUX", None);
+            }
```

Comment:
I'm happy to add `netbsd` and `openbsd` support. But `bitrig` seems dead? Last release was 6 years ago and no updates on github since 2017.

Reply:
Fair point! I thought I would add it for completeness. Removed now.